src/db/PgClient.ts

```diff
@@ -60,7 +75,7 @@ export default class PgClient implements ServerConnectionInterface {
     client.release();
     return Promise.all(
       result.rows.map((row: DBProcedure) => {
-        log.verbose(row.routine_name);
+        log.silly(row.routine_name);
```

Comment:
Does this need to be handled in `setLog()`?

Reply:
No, this change was just because initially I thought that debug was higher than verbose, but it's actually lower so this prints every single time. Changing to silly means we can choose to show it but it doesn't by default.